pom.xml

```diff
@@ -160,7 +160,7 @@
 
         <jre.version>1.8</jre.version>
         <httpcomponents.httpclient.version>4.5.13</httpcomponents.httpclient.version>
-        <httpcomponents.httpcore.version>4.4.11</httpcomponents.httpcore.version>
+        <httpcomponents.httpcore.version>4.4.13</httpcomponents.httpcore.version>
```

Comment:
Can we upgrade to the latest version 4.4.15?

https://downloads.apache.org/httpcomponents/httpcore/RELEASE_NOTES-4.4.x.txt

Reply:
Then we may also need to upgrade `org.apache.httpcomponents:httpclient` too as `org.apache.httpcomponents:httpclient:4.5.13` is depending on `org.apache.httpcomponents:httpcore:4.4.13` https://github.com/aws/aws-sdk-java-v2/issues/2927

Honestly I hope the aws-sdk-java-team can offer a complete solution to this issue in the future :)

Reply:
Looks like the latest version of `org.apache.httpcomponents:httpclient` in Maven is still 4.5.13. https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient  Does that mean the enforcer plugin will prevent users from using the latest httpcore simply because the latest httpclient does not depend on the latest httpcore? Hmm, wouldn't it defeat the purpose?

Anyway, we can merge this PR. As to a solution, I think this should be addressed on the Apache HTTP Client side, if they introduce something like [BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html), it would definitely help all consumers. For us, the only thing I can think of is to add a reminder note here to update httpcore whenever we update httpclient.

Reply:
I think the enforcer rule just detects version conflicts:) (not a maven expert). Thanks for taking a look!